test/validate_v2.ts

```diff
@@ -120,6 +120,18 @@ describe('Cards', () => {
       }
     });
   });
+
+  it('have valid subtypes', () => {
+    const subtypes = new Set<string>(getCardSubtypesV2Json().map(c => c.id));
+    cardFiles.forEach(file => {
+      const card = JSON.parse(fs.readFileSync(resolve(cardDir, file), 'utf-8'));
```

Comment:
I just realized, you're reading and parsing the files in every test? Maybe move this line onto a before (not beforeEach) block or even just directly into a top-level const.